swjsq.py

```diff
@@ -8,6 +8,13 @@
 import binascii
 import tarfile
 import atexit
+import socket
+
+originGetAddrInfo = socket.getaddrinfo
+def getAddrInfoWrapper(host, port, family=0, socktype=0, proto=0, flags=0):

```

Comment:
let's use getaddrinfo_wrapper here, and origin_getaddrinfo before to make the naming consistent in this file